mautic/mautic
# Fix segment membership report contact's columns

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
This just improve column naming of segment membership report. This make issue in french language where name and first name have the same translation string (Nóm).



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create segment membership report
3. See in columns after PR labels like Contact Firstname, Contact Lastname etc. Before were Firstname, Lastname
